### DISCUSSION TITLE
Oh boy, drama time!

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#93] [General (Friday Night Chattin')]
I went to the Psych Discussions just now, wondering how it’s going, only to see how much worse and how much of a fucking mess it is compared to the original discussions page, not to mention mayo is literally encouraging it by doing trolls. I used to see mayo as a cool, chill person, man, was I wrong.

## Now I can actually agree with ShadowMario taking down the discussions page

**Comment by Theoyeah** [comment]


**Reply by ThatOneSillyGuyDon** [reply]
> Fuck mayo, HE JUST BANNED ME FOR NO REASON !

I mean, he just made this poll rn

![9E276AAE-3187-4081-AA45-40D521EAA353](https://user-images.githubusercontent.com/87618246/167170837-6c5fe7d5-9df2-43ee-a7a7-2528223d8d7b.png)

(I voted for the 25 minutes one)

**Reply by ghost** [reply]
> > Fuck mayo, HE JUST BANNED ME FOR NO REASON !
> 
> I mean, he just made this poll rn
> 
> ![9E276AAE-3187-4081-AA45-40D521EAA353](https://user-images.githubusercontent.com/87618246/167170837-6c5fe7d5-9df2-43ee-a7a7-2528223d8d7b.png)
> 
> (I voted for the 25 minutes one)

no the checkmark means you requested to remove that one

**Reply by ThatOneSillyGuyDon** [reply]
> no the checkmark means you requested to remove that one

# what kind of fucking knowledge is that? That's like saying an Apple is a vegetable

**Reply by ghost** [reply]
> > no the checkmark means you requested to remove that one
> 
> # what kind of fucking knowledge is that? That's like saying an Apple is a vegetable

ITS NOT?

**Reply by megasssdev** [reply]
> ITS NOT?


Exactly

**Comment by megasssdev** [comment]
i havent been on there in weeks wtf happened

**Reply by ghost** [reply]
Demo agrees with ShadowMario on how discussions are shit.

**Comment by ghost** [comment]
-Facepalms
